woocommerce/woocommerce-gateway-stripe
# Fixes adding a new bancontact or iDeal payment method on payment method page.

Fixes adding a new bancontact or iDeal payment method on payment method page.
closes #1929

Turns out we are trying to hard to add a new payment method from a payment intent. We should just let the page load and ask Stripe if there are any new payment methods.

If we get the payment method from the setup intent id passed during the redirect it will be with the type bancontact or iDeal but if we query the customers account for all payment methods we can see the new payment method of the type sepa_debit that has the last 4 digits and a completely different payment method id. 

# Testing instructions
Go to payment methods. Try to add a new bancontact or ideal payment method. It should work.
Try again and force it to fail. It will fail with a relevant error message.
